screens/TimerScreen.js

```diff
@@ -69,6 +71,22 @@ const TimerScreen = ({ navigation }) => {
     return;
```

Comment:
dont need empty return, empty return should only be for when you wanna exit a function early, it will be the same whether or not you have it here